peazip-sources/res/share/lang/ru.txt

```diff
@@ -3,16 +3,16 @@
 10.3
 Перевод: Rytikov Slava, Dmitry Tsvetkov, xoxmodav, ViktorOn , Aleksandr (yupiter43@gmail.com)
 Посл. пров.: ViktorOn
-Посл. ред.: 02.02.2025
+Посл. ред.: 25.02.2025
 
 === PeaZip text group ===
 
-txt_10_3_classic: Classic
-txt_10_3_ascii: It is recommended to use only ASCII characters in passwords for the current archive format (mixed case letters, numbers, and special characters).
-txt_10_3_minimal: Minimal
-txt_10_3_m: Modern
-txt_10_3_pm: Postmodern
-txt_10_3_aw: Workaround: check "Force typing password interactively" in the password prompt, or set Console mode for backend binaries.
+txt_10_3_classic: Классический
+txt_10_3_ascii: Рекомендуется использовать только ASCII-символы в паролях для текущего формата архива (буквы в верхнем и нижнем регистре, цифры и спецсимволы).
+txt_10_3_minimal: Минималистичный
+txt_10_3_m: Модерн
+txt_10_3_pm: Постмодерн
+txt_10_3_aw: Обходной путь: установите флажок «Принудительный ввод паролей в интерактивном режиме» в запросе пароля или включите консольный режим для исполняемых файлов бэкенд.
 txt_10_2_savecopy: Сохранить копию
 txt_10_1_full: Full

```

Comment:
I would be glad to receive a description of this string, thank you @peazip

Reply:
txt_10_1_full: Full
Is used in Optios > Settings > Themes, Breadcrumb bar decoration (second dropdown).
Options are None (breadcrumb is only colorized as in first dropdown menu) Underlined (breadcumb is colorized and underlined) and Full, in which the breadcrumb is colorized and fully encircled by a continuos line.

Reply:
Maybe I'm missing something, but I only see a difference between None and Underlined/Full (together) 🤔

Reply:
With Full you should see a very light line around the breadcrumb, plus a  comparably darker "shadow" on the bottom.
With Underlined you should see only the "shadow" on the bottom.
Using a different color for filling the breadcrumb area (first parameter) can make those lines less or more easily visible.